docs/dev/building-on-zksync/hello-world.md

```diff
@@ -27,7 +27,7 @@ This is what we're going to do:
   - [Alchemy Goerli faucet](https://goerlifaucet.com/)
   - [Paradigm Goerli faucet](https://faucet.paradigm.xyz/)
   - [Proof of work faucet](https://goerli-faucet.pk910.de/)
-- ERC20 tokens on zkSync are required for the testnet paymaster. We recommend using [the faucet from the zkSync portal](https://goerli.portal.zksync.io/faucet).
+- ERC20 tokens on zkSync are required for the testnet paymaster. Get testnet `ETH` on zkSync Era using [bridges](https://zksync.io/explore#bridges) to bridge funds to zkSync.
```

Comment:
"on"-> "for"

Reply:
Ah, and given we are talking about testnet tokens, people will need to swap them on some dex. Worth pointing them to some, or point them directly to the token contracts (assuming they can mint directly)

Reply:
Fixed